package.json

```diff
@@ -29,6 +29,7 @@
     "bootstrap": "4.2.1",
     "d3": "5.7.0",
     "font-awesome": "4.7.0",
+    "fuse.js": "3.3.0",
```

Comment:
This library seems pretty neat:
http://fusejs.io/

...and is small enough that it should be fine to use:
https://bundlephobia.com/result?p=fuse.js@3.3.0

@camd, I wonder if this library could be used to simplify the jobs-view filter implementation at a later point? :-)

Reply:
Yeah, looks interesting.  I'll check it out.  :)